addons/web_editor/static/src/js/backend/field_html.js

```diff
@@ -309,6 +309,7 @@ var FieldHtml = basic_fields.DebouncedField.extend(TranslatableFieldMixin, {
                 }
                 self.$content = $('#iframe_target', self.$iframe[0].contentWindow.document.body);
                 resolver();
+                core.bus.trigger('iframe_updated', self.$iframe);
```

Comment:
Could also be in a mass_mailing "_toggleCodeView" override since the iframe_updated thing is only used in mass_mailing anyway.

Reply:
Right now that's the only place we use it but if we're going to trigger an event to notify whenever the iframe is updated, we might as well do it every time the iframe is updated, for future proofinessitude.